### DISCUSSION TITLE
Error on Test Connection

### DISCUSSION THREAD

**Discussion by valencio2** [dbeaver/dbeaver#20675] [Q&A]
### Description

I'm trying to install DBeaver but when I make the configuration steps, that shows me this error. 
<img width="1435" alt="Screenshot 2023-07-24 at 09 09 23" src="https://github.com/dbeaver/dbeaver/assets/140404183/ebcbb57e-5043-429b-86bf-d07b3160e716">

### DBeaver Version

DBeaver Community 23.1.3

### Operating System

macOS Intel Core i5

**Comment by E1izabeth** [comment]
Please add more details. What database are you trying to connect to? How exactly are you configuring connection?

**Comment by valencio2** [comment]
I’m trying to connect on this database: MWNCDSNB

I’m following these steps to configurate: https://ibm.ent.box.com/s/rfhe2s94greayd4mt1tkabz0l1zx4a2k

**Comment by E1izabeth** [comment]
Screenshots doesn't come to github if you reply on a notification email. Please, attach them in a comment to the issue directly.

**Comment by valencio2** [comment]
Sorry. Here you go.

<img width="1080" alt="Screenshot 2023-07-24 at 09 18 21" src="https://github.com/dbeaver/dbeaver/assets/140404183/d8c518e6-209e-4555-a67c-17790e7968da">

**Comment by LonwoLonwo** [comment]
Hello @valencio2 

On the assumption of the error message from the first screen - you need to specify the SSL Trust store file path in the sslTrustStoreLocation property.
Like:
`Security/SSL/certificates/myTrustStore.jks`

**Comment by valencio2** [comment]
How could I get this info in Mac? Could you please help me with this?

Isabella Valêncio Bueno
Workforce Staffing Professional
IBM Consulting - Brazil Workforce Management
19 99160 1991 (Mobile)

**Comment by E1izabeth** [comment]
You put the path to the directory containing the file instead of the path to the file in `sslTrustStoreLocation` driver property. It's not something mac-specific, just put a valid path there.

**Comment by valencio2** [comment]
Ok, but my question is: how could I get the information of path to the directory? Is in “Get info” > “where” on Mac?

**Comment by E1izabeth** [comment]
I can't help you there as I'm not a mac user, but you could google it.

**Comment by valencio2** [comment]
This problem is solved, but now we have another error
<img width="1292" alt="Screenshot 2023-07-24 at 14 13 39" src="https://github.com/dbeaver/dbeaver/assets/140404183/7a0767ea-b87d-4281-b58d-700054728417">

**Comment by E1izabeth** [comment]
You need to contact your database administrator as your password has been expired.

**Comment by valencio2** [comment]
Yes, I contact him. I have here another temporary passphrase and the guide is that I need to change the password inside DBeaver.

**Comment by valencio2** [comment]
If you're unable to use the passphrase due to an application and/or
 system restriction use CLAS option 1.1.1 to request a password.
 
 NOTE:  When you logon to the system and type in the above temporary
 passphrase a message may be displayed: "The Passphrase has EXPIRED",
 this is a normal RACF message which requires you to type in a new
 passphrase. After typing a new passphrase, RACF asks you to verify
 the passphrase you just typed in, so you need to type it in again.
